Item Requirements Plugin

AhDoozy opened the pull request:
The Item Requirements Plugin is your in-game crystal ball for item eligibility. Whether you’re browsing your bank, sorting your inventory, or daydreaming in the Grand Exchange, this plugin automatically spots items you can’t use yet, whether they’re equipment, consumables, or other special items—then tells you exactly why.

riktenx commented:
Invalid submission, see https://github.com/runelite/plugin-hub?tab=readme-ov-file#submitting-a-plugin.

riktenx commented:
Submission is valid now, please check build failure as it seems you're using a few forbidden APIs.

LlemonDuck commented:
* remove the empty config
* unless the screenshots are used in the plugin itself at runtime, remove them from the resources dir

AhDoozy commented:
> * remove the empty config
> * unless the screenshots are used in the plugin itself at runtime, remove them from the resources dir

I've removed the empty config.java file and moved the screenshots folder out of the resources directory.

AhDoozy commented:
ping for review.

pajlada commented:
> ping for review.

@AhDoozy just so you know, this is not how it works
